os/kernel/clock/clock_systimer.c

```diff
@@ -126,16 +126,22 @@ clock_t clock_systimer(void)
 #endif							/* CONFIG_SYSTEM_TIME64 */
 #else							/* CONFIG_SCHED_TICKLESS */
 #ifdef CONFIG_SYSTEM_TIME64
+        uint32_t high,low,high_check;
+        volatile uint32_t *addr;
+
+        addr = &g_system_timer;
+ 
+        high = addr[1];
+        do
+        {
+          high_check = high;
+           low = addr[0];
+           high = addr[1];
+
+        }
+        while(high != high_check);
```

Comment:
```suggestion
  do {
    high_check = high;
    low = addr[0];
    high = addr[1];
  } while(high != high_check);
```

please check this coding rule.

the indentation is 4 size tab